sql/core/src/test/resources/sql-tests/results/subquery/negative-cases/invalid-correlation.sql.out

```diff
@@ -63,7 +63,7 @@ WHERE  t1a IN (SELECT   min(t2a)
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-Resolved attribute(s) t2b#x missing from min(t2a)#x,t2c#x in operator !Filter t2c#x IN (list#x [t2b#x]).
+cannot resolve 't2b' given input columns: [min(t2a), t2.t2c]; line 6 pos 24
```

Comment:
Neither of the error messages is good, we can improve it later when we improve the correlated subquery resolution.